autoscraper/auto_scraper.py

```diff
@@ -99,24 +94,17 @@ def build(self, url=None, wanted_list=None, html=None, request_args=None):
                 result_list += result
                 stack_list.append(stack)
 
-        result_list = self.unique(result_list)
+        result_list = unique(result_list)
 
-        if self._check_result(result_list, wanted_list):
-            self.stack_list = self.unique(stack_list)
+        if forall(lambda x: x in wanted_list, result_list):
```

Comment:
Why not using `all` builtin function?
`all(w in result for w in wanted_list)`
Then we don't need to declare the `forall` function.

Reply:
I thought of performance penalty... totally forgot it makes generator without brackets...